alacritty_terminal/src/selection.rs

```diff
@@ -214,16 +214,49 @@ impl Selection {
         span.map(|mut span| {
```

Comment:
What other apps do with block selection when on one line you have 2 full width glyphs?

Anyway, I feel like you have a bug with block selection and 2 width chars. All selection on this gif was block one, I hope this gif gives enough explanation, if it's not feel free to ask.

![block](https://user-images.githubusercontent.com/27620401/70935274-e1716900-2050-11ea-8536-6c030ffbd655.gif)

Reply:
We only ever check the beginning or end of a block selection, that's why the bottom left and top right will not snap to the full width glyphs. I'm not sure if it's worth fixing this, or if we maybe should just disable this completely for block selection.

I'll check what VTE does maybe that can give some insights..

Reply:
I'm voting for consistency here. So we shouldn't do anything where we're in a block selection or do the same thing as vte(expand selectively on full width chars).

Reply:
This is unrelated to this PR, I'll create a follow-up issue about this.

Reply:
It turned out that this thing is completely unrelated to this PR, so it should be moved to our issue tracker instead.

Reply:
I think we've reach agreement that we should do the same thing as VTE here (expand full width in block selections), however that is outside the scope of this PR.